src/app/devices/sol/sol.component.ts

```diff
@@ -74,14 +109,31 @@ export class SolComponent implements OnInit {
       cursorStyle: 'block',
       fontWeight: 'bold'
     })
+    this.term.open(this.container)
+    this.term.onData((data: any) => {
+      this.handleKeyPress(data)
+    })
+    this.term.attachCustomKeyEventHandler((e: any) => {
+      e.stopPropagation()
+      e.preventDefault()
+      if (e.ctrlKey && e.shiftKey && e.keyCode === 67) {
```

Comment:
leverage @angular/cdk/keycodes for constants (https://github.com/angular/components/blob/master/src/cdk/keycodes/keycodes.ts)  should be C

Reply:
done